deepspeed/inference/engine.py

```diff
@@ -181,6 +237,11 @@ def remove_mask_prepare_for_bloom(self):
             if hasattr(self.module.transformer, '_prepare_attn_mask'):
                 self.module.transformer._prepare_attn_mask = lambda attention_mask, *args, **kwargs: attention_mask
 
+    def build_alibi_tensor(self):
```

Comment:
it will has still alibi issue if user are not using the latest huggingface which may contain https://github.com/intel-sandbox/transformers/pull/8, so suggest the right merge sequence is huggingface merge the change firstly. then MS merge the corresponding change in deepspeed.